flexmeasures/data/schemas/reporting/aggregation.py

```diff
@@ -0,0 +1,65 @@
+from marshmallow import fields, ValidationError, validates_schema
+
+from flexmeasures.data.schemas.reporting import ReporterConfigSchema
+
+from enum import Enum
+
+
+class AggregationMethod(Enum):
+    SUM = "SUM"
+    MEAN = "MEAN"
+
+
+class AggregatorSchema(ReporterConfigSchema):
+    """Schema for the reporter_config of the AggregatorReporter
+
+    Example:
+    .. code-block:: json
+        {
+            "beliefs_search_configs": [
+                {
+                    "sensor": 1,
+                    "source" : 1,
+                    "alias" : "PV"
+                },
+                {
+                    "sensor": 1,
+                    "source" : 2,
+                    "alias" : "CONSUMPTION"
+                }
+            ],
+            "method" : "SUM",
+            "weights" : {
+                "PV" : 1.0,
+                "CONSUMPTION" : -1.0
+            }
+        }
+    """
+
+    method = fields.Enum(AggregationMethod, required=True)
```

Comment:
Should we set the default to "MEAN" and not make it required?

Reply:
Sure! I suggest to default to "SUM", though. I feel we are going to use this Reporter a lot to aggregate energy quantities.